### PR TITLE
Add Vivek Jha Profile

### PR DESCRIPTION
**Pull request by 8rxn**
**Description**

This PR fixes [#4254](https://github.com/layer5io/layer5/issues/4254) 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 78352ef51b9f2c84dd333296712344555a935afb at: https://6469036ee990d546e8eaabc3--layer5.netlify.app

**Comment by leecalcote** [comment]
@duckling69

**Comment by duckling69** [comment]
> @duckling69

Looks good 
Thanks @8rxn 🤠

File: src/collections/members/vivek-jha/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: Vivek Jha
+position: Contributor
+image_path: ./vivek-jha.png
+github: duckling69
+twitter: vivekjha30
+linkedin: vivek-jha-17000918b
+location: New Delhi, India
+bio: Hi I'm a undergrad student at NIT Rourkela who loves to explore new technologies currently diving into world of open-source. I'm proficient with C/C++ , JavaScript, React, SQL, Bash and python. I'm currently exploring cloud native and DevOps and learning golang, CI/CD pipelines, docker, kubernetes. Always up for any kind of collaboration Feel free to connect on any of my social handles.
+badges: 
+ - community
```
Comment (Chadha93) [inline src/collections/members/vivek-jha/index.mdx:11]:
```suggestion
```

**Comment by l5io** [comment]
🚀 Preview for commit 53fc6e975359710df25602221e5fdcbc1c571d5e at: https://646b062328956c1dbf64f544--layer5.netlify.app

**Comment by 8rxn** [comment]
Thanks for making the change 🤠 @Chadha93

**Review by Chadha93 — APPROVED** [review]

**Comment by Waltvinny** [comment]
🥰🥰

**Pull request merged**